src/UI/Component/Input/Field/Factory.php

```diff
@@ -348,5 +347,41 @@ public function password($label, $byline = null);
 	 * @return \ILIAS\UI\Component\Input\Field\Select
 	 */
 	public function select($label, array $options, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A multi-select is used to allow users to pick several options from a list.
+	 *   composition: >
+	 *     The multi-select field will render labeled checkboxes according to given options.
+	 *   effect: >
+	 *
+	 *   rivals:
+	 *     Checkbox field: Use a checkbox field for a binary yes/no choice.
+	 *     Tag field: Use a tag input when the user is able to extend the list of given options.
+	 *     Select field: Use a select input when the user's choice is limited to one option.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *      A multi-select input SHOULD be used when a user has to choose from a finite list of options
+	 *      which cannot be extended by the user's input and where more than one choice can be made.
+	 *     2: >
+	 *      A multi-select input MUST NOT be used whenever a user has to perform a binary choice where
+	 *      option is automatically the inverse of the other. A Checkbox MUST be used in this case.
+	 *     3: >
+	 *      A multi-select input MUST NOT be used whenever a user has to perform a choice from a list of
+	 *      options where only one option can be selected. A Select MUST be used in this case
+	 *
```

Comment:
wording:
- Also add here the only postive... rule you proposed for the Radio Input